src/__experimental__/components/input/input-presentation.component.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```

Comment:
What do you think about to move every file related with `input-presentation` to external folder? It's going to make it more readable

Reply:
We are trying a flatter structure for directories, see the wiki https://github.com/Sage/carbon/wiki/Codebase-Conventions#multiple-components

We will keep it as it is for now, but if we find it gets unmanageable we can update later.